packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -59,6 +59,7 @@ const propTypes = {
 
   /**
    * Child element to be displayed on the right end of the header.
+   * This is intended to be used with the CollapsibleMenuView.
```

Comment:
your source code doesn't match the lib code, not sure if it matters if you don't recompile though but would be nice